src/vaillant/15.basv3.tsp

```diff
@@ -0,0 +1,3286 @@
+    @minValue(0) @maxValue(90) @step(1)
+    value: tempv;
+  }
+
+  /** SetbackMode heating circuit 1 */
+  @inherit(r_3, w_3)
+  @ext(0xe, 0)
+  @condition(Basv3.Hc1CircuitType.value, "1")
+  model Hc1SetbackMode {
+    value: offmode2;
+  }
+
+  /** HeatCurve heating circuit 1 */
+  @inherit(r_3, w_3)
+  @ext(0xf, 0)
+  model Hc1HeatCurve {
+    @minValue(0.1) @maxValue(4) @step(0.05)
+    value: EXP;
+  }
+
+  /** HeatingMaxFlowTempDesired heating circuit 1 */
+  @inherit(r_3, w_3)
+  @ext(0x10, 0)
+  model Hc1HeatingFlowTempMax {
+    @minValue(15) @maxValue(90) @step(1)
+    value: tempv;
+  }
+
+  /** CoolingMinFlowTempDesired heating circuit 1 */
+  @inherit(r_3, w_3)
+  @ext(0x11, 0)
+  @condition(Basv3.Hc1CircuitType.value, "1")
+  @condition(Basv3.Hc1CoolingEnabled.value, "1")
+  model Hc1CoolingFlowTempMin {
+    @minValue(7) @maxValue(24) @step(1)
+    value: tempv;
+  }
+
+  /** HeatingMinFlowTempDesired heating circuit 1 */
+  @inherit(r_3, w_3)
+  @ext(0x12, 0)
+  model Hc1HeatingFlowTempMin {
+    @minValue(15) @maxValue(90) @step(1)
+    value: tempv;
+  }
+
+  /** ExtHwcOpMode heating circuit 1 */
+  @inherit(r_3, w_3)
+  @ext(0x13, 0)
+  @condition(Basv3.Hc1ExtHwcActive.value, "1")
+  model Hc1ExtHwcOpMode {
+    value: opmode2;
+  }
+
+  /** summer outside switchoff temperature heating circuit 1 */
+  @inherit(r_3, w_3)
+  @ext(0x14, 0)
+  model Hc1SummerTempLimit {
+    @minValue(10) @maxValue(99) @step(1)
+    value: tempv;
+  }
+
+  /** RoomTempMod heating circuit 1 */
+  @inherit(r_3, w_3)
+  @ext(0x15, 0)
+  model Hc1RoomTempMod {
+    value: rcmode2;
+  }
+
+  /** ExtHwcActive heating circuit 1 */
+  @inherit(r_3)
+  @ext(0x18, 0)
+  model Hc1ExtHwcActive {
+    value: onoff;
+  }
+
+  /** ExtHeatDemand heating circuit 1 */
+  @inherit(r_3)
+  @ext(0x19, 0)
+  model Hc1ExtHeatDemand {
+    value: onoff;
+  }
+
+  /** MixerMovement heating circuit 1 */
+  @inherit(r_3)
+  @ext(0x1a, 0)
+  @condition(Basv3.Hc1CircuitType.value, "1")
+  model Hc1MixerMovement {
+    value: EXP;
+  }
+
+  /** HeatCurveAdaption heating circuit 1 */
+  @inherit(r_3)
+  @ext(0x1c, 0)
+  model Hc1HeatCurveAdaption {
+    value: EXP;
+  }
+
+  /** Status heating circuit 1 */
+  @inherit(r_3, w_3)
+  @ext(0x1b, 0)
+  model Hc1Status {
+    value: onoff;
+  }
+
+  /** PumpStatus heating circuit 1 */
+  @inherit(r_3)
+  @ext(0x1e, 0)
+  model Hc1PumpStatus {
+    value: hcpumpmode;
+  }
+
+  // ##### heating circuit 2 #####
+
+  /** default *r */
+  @base(MF, 0x24, 0x2, 0, 0x2, 1)
+  model r_4 {
+    @maxLength(4)
+    value: IGN;
+  }
+
+  /** default *w */
+  @write
+  @base(MF, 0x24, 0x2, 1, 0x2, 1)
+  model w_4 {}
+
+  /** CircuitType heating circuit 2 */
+  @inherit(r_4)
+  @ext(0x2, 0)
+  model Hc2CircuitType {
+    value: hctype;
+    @maxLength(1) ign: IGN;
+  }
+
+  /** DesiredReturnTemp heating circuit 2 */
+  @inherit(r_4, w_4)
+  @ext(0x4, 0)
+  @condition(Basv3.Hc1CircuitType.value, "4")
+  model Hc2DesiredReturnTemp {
```

Comment:
sure this is return temp and not flow temp? wondering what a controller would do with that one instead of the flow temp - adjust the mixer to have the room cooling down/heating up just to reach the target return temp?